File: src/viash/skeleton/template_script.py
print("This is a skeleton component")
print("The arguments are:")
print(" - input: ", par["input"])
print(" - output: ", par["output"])
print(" - option: ", par["option"])
print("")


with open(par["input"], "r") as reader, open(par["output"], "w") as writer:
    lines = reader.readlines()
    
    new_lines = [par["option"] + x for x in lines]
    
    writer.writelines(new_lines)

File: src/viash/skeleton/template_script.R
cat("This is a skeleton component\n")
cat("The arguments are:\n")
cat(" - input: ", par$input, "\n", sep = "")
cat(" - output: ", par$output, "\n", sep = "")
cat(" - option: ", par$option, "\n", sep = "")
cat("\n")

cat("Reading input file\n")
lines <- readLines(par$input)

cat("Running output algorithm\n")
new_lines <- paste0(par$option, "-", lines)

cat("Writing output file\n")
writeLines(new_lines, con = par$output)
